doc/source/api/visualization/index.rst

```diff
@@ -6,106 +6,65 @@ Visualization
-  solver_session.tui.display.objects.contour['contour-1']()
-  solver_session.tui.display.objects.contour['contour-1'].field.set_state('velocity-magnitude')
-  solver_session.tui.display.objects.contour['contour-1'].field()
-  solver_session.tui.display.objects.contour['contour-1'].color_map.size.set_state(80.0)
-  solver_session.tui.display.objects.contour['contour-1'].color_map.size()
-  solver_session.tui.display.objects.contour['contour-1'].rename('my-contour')
-  del solver_session.tui.display.objects.contour['my-contour']
+  solver_session.settings.results.graphics.contour['contour-1'] = {'boundary_values': True, 'color_map': {'color': 'field-velocity', 'font_automatic': True, 'font_name': 'Helvetica', 'font_size': 0.032, 'format': '%0.2e', 'length': 0.54, 'log_scale': False, 'position': 1, 'show_all': True, 'size': 100, 'user_skip': 9, 'visible': True, 'width': 6.0}, 'coloring': {'smooth': False}, 'contour_lines': False, 'display_state_name': 'None', 'draw_mesh': False, 'field': 'pressure', 'filled': True, 'mesh_object': '', 'node_values': True, 'range_option': {'auto_range_on': {'global_range': True}}, 'surfaces': [2, 5]}
+  solver_session.settings.results.graphics.contour['contour-1']()
+  solver_session.settings.results.graphics.contour['contour-1'].field = 'velocity-magnitude'
+  solver_session.settings.results.graphics.contour['contour-1'].field()
+  solver_session.settings.results.graphics.contour['contour-1'].color_map.size.set_state(80.0)
+  solver_session.settings.results.graphics.contour['contour-1'].color_map.size()
+  solver_session.settings.results.graphics.contour['contour-1'].rename('my-contour')
+  del solver_session.settings.results.graphics.contour['contour-1']['my-contour']
 
-PyVista example (graphics)
---------------------------
+Visualization example
+---------------------
 
 Here field data is extracted from the Fluent session into the Python
-environment. PyVista is then used to visualize the extracted data.
+environment. Visualization is then used to visualize the extracted data.
 
-.. code:: python
-
-  # import module
-  from ansys.fluent.visualization import Graphics
-
-  # get the graphics objects for the session
-
-  graphics_session1 = Graphics(solver_session)
-  mesh1 = graphics_session1.Meshes["mesh-1"]
-  contour1 = graphics_session1.Contours["contour-1"]
-  contour2 = graphics_session1.Contours["contour-2"]
-  surface1 = graphics_session1.Surfaces["surface-1"]
-
-  # set graphics objects properties
-
-  # mesh
-  mesh1.show_edges = True
-  mesh1.surfaces = ['symmetry']
+.. code-block:: python
 
-  # contour
-  contour1.field = "velocity-magnitude"
-  contour1.surfaces = ['symmetry']
+      from ansys.fluent.visualization import GraphicsWindow, Mesh, Contour, Surface, XYPlot
 
-  contour2.field = "temperature"
-  contour2.surfaces = ['symmetry', 'wall']
+      # get the graphics objects for the session
```

Comment:
I think we are creating new objects, not accessing exisitng ones.

Regarding the code-comments, it is better to break the large code-blocks like this and move the comments outside the code-block to have them in RST text format.